lib/rules/font-weight-notation/__tests__/index.js

```diff
@@ -320,3 +320,15 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  syntax: "scss",
+  config: ["numeric", { ignoreFunctions: ["map-deep-get"] }],
+
+  accept: [
+    {
+      code: "a { font-weight: map-deep-get($theme, font, weight, normal); }"
+    }
+  ]
```

Comment:
Please add a couple of `reject` tests e.g. 

`code: "a { font-weight: bold }`
`code: "a { font: italic my-function($var) 20px sans-serif;`